### PR TITLE
chore: upgrade gatling-enterprise-plugin-commons 1.9.0-M5

### PR DESCRIPTION
**Pull request by slandelle**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -9,8 +9,7 @@ githubPath := "gatling/gatling-sbt-plugin"
 
 libraryDependencies ++= Seq(
   "org.scalatest"     %% "scalatest"                         % "3.2.17" % Test,
-  "org.zeroturnaround" % "zt-zip"                            % "1.16",
-  "io.gatling"         % "gatling-enterprise-plugin-commons" % "1.8.0"
+  "io.gatling"         % "gatling-enterprise-plugin-commons" % "1.9.0-M5"
```
Comment (Isammoc) [inline build.sbt]: Need format

`sbt scalafmtSbt`

File: src/main/scala/io/gatling/sbt/settings/gatling/TaskEnterprisePackage.scala [hunk 2]
```diff
@@ -18,11 +18,20 @@ package io.gatling.sbt.settings.gatling
 
 import java.io.File
 
+import io.gatling.plugin.io.PluginLogger
+import io.gatling.plugin.pkg.EnterprisePackager
 import io.gatling.sbt.settings.gatling.EnterpriseUtils.InitializeTask
 import io.gatling.sbt.utils._
 
+import collection.JavaConverters.*
```
Comment (Isammoc) [inline src/main/scala/io/gatling/sbt/settings/gatling/TaskEnterprisePackage.scala:26]:
```suggestion
import collection.JavaConverters._
```

File: src/main/scala/io/gatling/sbt/settings/gatling/TaskEnterprisePackage.scala [hunk 3]
```diff
@@ -18,11 +18,20 @@ package io.gatling.sbt.settings.gatling
...
+
+import sbt._
 import sbt.{ Configuration, Def, IntegrationTest, ModuleDescriptorConfiguration, Test }
-import sbt.Keys._
+import sbt.Keys.*
```
Comment (Isammoc) [inline src/main/scala/io/gatling/sbt/settings/gatling/TaskEnterprisePackage.scala:30]:
```suggestion
import sbt.Keys._
```

File: src/main/scala/io/gatling/sbt/utils/DependenciesAnalyzer.scala [hunk 4]
```diff
@@ -64,27 +58,40 @@ object DependenciesAnalyzer {
...
-    val excludedDeps = configurationReport.modules.filterNot(isTransitiveGatlingDependency(_, callers))
 
-    val nonGatlingDependencies = excludedDeps
-      .flatMap(_.artifacts)
-      .collect { case (artifact, file) if artifact.`type` == Artifact.DefaultType || artifact.`type` == "bundle" => file }
+    val allModules = configurationReport.modules.toSet
 
-    DependenciesAnalysisResult(gatlingVersion, nonGatlingDependencies)
+    val gatlingModules = allModules.filter(isTransitiveGatlingDependency(_, callers))
+    val nonGatlingModules = allModules -- gatlingModules
```
Comment (Isammoc) [inline src/main/scala/io/gatling/sbt/utils/DependenciesAnalyzer.scala:72]:
```suggestion
    val (gatlingModules, nonGatlingModules) = allModules.partition(isTransitiveGatlingDependency(_, callers))
```

Comment (slandelle) [reply src/main/scala/io/gatling/sbt/utils/DependenciesAnalyzer.scala:72]: facepalm

File: src/main/scala/io/gatling/sbt/settings/gatling/TaskEnterprisePackage.scala (hunk 2 above)
Comment (Isammoc) [reply src/main/scala/io/gatling/sbt/settings/gatling/TaskEnterprisePackage.scala:26]: Still here

Comment (slandelle) [reply src/main/scala/io/gatling/sbt/settings/gatling/TaskEnterprisePackage.scala]: done

**Pull request merged**